tst/regression/mocmc_diffusion.gold

```diff
@@ -1,25 +1,25 @@
-2.578814481287522261e-07
-8.539726882840010634e-11
-9.881445129638454866e-11
-2.880460357023297414e-08
-2.198153555894490595e-03
-6.512936231879706828e-01
-1.589897079949697579e-04
-2.487714603862677815e-02
-2.197674042014615206e-02
+2.578814481287529143e-07
```

Comment:
These upgolds are probably due to the changed numerics of the radiation source rootfind

Reply:
do the answers still look right?

Reply:
yep, they differ at like the 10th decimal place or something.